### ISSUE TITLE
License issue

### ISSUE THREAD

**Issue by isaacs** [jonschlinkert/parse-author#4]
Hi.  It looks like [this code](https://github.com/jonschlinkert/parse-author/blob/master/index.js#L10-L22) was copied from https://github.com/npm/normalize-package-data/blob/master/lib/fixer.js#L359-L369

That in itself is fine, but only if the terms in the BSD-2-Clause license are followed.

Can you please comply with the conditions in the license?  https://github.com/npm/normalize-package-data/blob/master/LICENSE

Specifically, include the copyright, list of conditions, and disclaimer in the LICENSE file and in the code, rather than a different copyright and license.  Without permission from the copyright holders (ie, Meryn Stol and Contributors), sublicensing is not allowed.

Thanks.

**Comment by jonschlinkert** [comment]
This library started with https://github.com/jonschlinkert/parse-authors/commit/e8f4b1107f66f07749ad97329409287ad384b18f. (related https://github.com/jonschlinkert/author-regex) Looks like a contributor changed the code to be more similar to what's in normalize-package-data in [this pull request](https://github.com/jonschlinkert/parse-author/pull/3). 

Thanks for bringing this to my attention. I'll take immediate action and re-write the code to something less brittle. Problem solved. 

Fwiw, it's good to see that you started caring about protecting other people's rights since you copied my code into minimatch without attribution.

**Comment by isaacs** [comment]
Thanks.

> you copied my code into minimatch without attribution.

That is untrue.

**Comment by notslang** [comment]
> Looks like a contributor changed the code to be more similar to what's in normalize-package-data in this pull request.

Sorry about that PR - I wasn't trying to cause any license issues. I just based the functionality that I added in 00e3cfc2c0bcabc02ae88d0a1471abd67248b34c and e590c5c4b8f1afd7c3ce38ddd35ec5305a35210a on the way npm parses author strings so I could standardize the behaviour with one module and use it in fixpack.

Idk if that makes it a derivative work or not, but this was 11 months ago, so I don't even remember if I checked for an attribution clause in the license.
